assets/helpers/tracking/googleTagManager.js

```diff
@@ -161,7 +187,7 @@ function init(participations: Participations) {
 }
 
 function successfulConversion(participations: Participations) {
-  pushToDataLayer('SuccessfulConversion', participations);
+  sendData('SuccessfulConversion', participations);
```

Comment:
Are we still using `pushToDataLayer`?, if not I would delete it.

Reply:
We only need it once per page load so it's being fired from `dataLayerReady` but we don't need it for everything else.